### PR TITLE
feature:added person class and person test

### PR DESCRIPTION
**Pull request by Kenrixkk22**

### CODE REVIEW

**Review by anthonydmays — APPROVED** [review]

File: lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/kenricksutherland/Persontest.java
Comment (anthonydmays) [inline lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/kenricksutherland/Persontest.java:1]: Please rename file to `PersonTest.java`.

Comment (anthonydmays) [inline lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/kenricksutherland/Persontest.java:1]: Run `./gradlew spotlessApply` to fix the formatting (in the `lesson_07/objects` directory of course).

**Review by anthonydmays — CHANGES_REQUESTED** [review]
Almost there! Fix the couple of issues and I'll get this submitted.

File: lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/kenricksutherland/Persontest.java
Comment (Kenrixkk22) [reply lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/kenricksutherland/Persontest.java:1]: changed named to person test

Comment (Kenrixkk22) [reply lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/kenricksutherland/Persontest.java:1]: ran those two commands

**Review by anthonydmays — APPROVED** [review]

**Pull request merged**